samples/client/petstore/typescript-axios/builds/composed-schemas/api.ts

```diff
@@ -79,16 +79,14 @@ export interface Dog {
     'breed'?: DogBreedEnum;
 }
 
-/**
-    * @export
-    * @enum {string}
-    */
-export enum DogBreedEnum {
-    Dingo = 'Dingo',
-    Husky = 'Husky',
-    Retriever = 'Retriever',
-    Shepherd = 'Shepherd'
-}
+export const DogBreedEnum = {
+    Dingo: 'Dingo',
```

Comment:
Please also add samples with stringEnums=true

Reply:
Sorry, I forget to add samples.

Reply:
I added samples with the name `with-string-enums`.